website/docs/r/report_rule.html.markdown

```diff
@@ -0,0 +1,138 @@
+---
+subcategory: "Report Rules"
+layout: "lacework"
+page_title: "Lacework: lacework_report_rule"
+description: |-
+  Create and manage Lacework Report Rules
+---
+
+# lacework\_report\_rule
+
+Use this resource to create a Lacework Report Rule in order to route reports to an email alert channel.
+For more information, see the [Report Rules documentation](https://docs.lacework.com/report-rules).
+
+## Example Usage
+
+#### Report Rule with Aws Compliance Reports
+```hcl
+resource "lacework_report_channel_email" "team_email" {
+  name       = "Team Emails"
+  recipients = ["foo@example.com", "bar@example.com"]
+}
+
+resource "lacework_report_rule" "example" {
+  name                 = "My Report Rule"
+  description          = "This is an example report rule"
+  email_alert_channels = [lacework_report_channel_email.team_email.id]
+  severities           = ["Critical", "High"]
+
+  aws_compliance_reports {
+    cis_s3 = true
+  }
+  weekly_snapshot = true
+}
+```
+
+#### Report Rule with Gcp Compliance Reports and Gcp Resource Group
+```hcl
+resource "lacework_report_channel_email" "team_email" {
+  name       = "Team Emails"
+  recipients = ["foo@example.com", "bar@example.com"]
+}
+
+resource "lacework_resource_group_gcp" "all_gcp_projects" {
+  name         = "GCP Resource Group"
+  description  = "All Gcp Projects"
+  organization = "MyGcpOrg"
+  projects     = ["*"]
+}
+
+resource "lacework_report_rule" "example" {
+  name             = "My Report Rule"
+  description      = "This is an example report rule"
+  channels         = [lacework_report_channel_slack.ops_critical.id]
+  resource_groups  = [lacework_resource_group_gcp.all_gcp_projects.id]
+  severities       = ["Critical"]
+
+  gcp_compliance_reports {
+    k8s = true
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The report rule name.
+* `channels` - (Required) The list of report channels for the rule to use.
```

Comment:
did you mean alert channels, also, they are email alert channels

Reply:
it should be email alert channels. Good catch!